NitroxClient/MonoBehaviours/DiscordRP/AcceptRequest.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections;
+using NitroxClient.MonoBehaviours.DiscordRP;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours.Gui.MainMenu
+{
+    public class AcceptRequest : MonoBehaviour
+    {
+        public DiscordController DiscordRp;
+        public DiscordRpc.DiscordUser Request;
+
+        Rect discordWindowRect = new Rect(Screen.width / 2 - 250, 100, 500, 250);
+        Texture avatar;
+
+        public void Awake()
+        {
+            DontDestroyOnLoad(gameObject);
+        }
+
+        public void Start()
+        {
+            DiscordRp = gameObject.GetComponent<DiscordController>();
+            StartCoroutine(LoadAvatar(Request.userId, Request.avatar));
+        }
+
+        public void OnGUI()
+        {
+            discordWindowRect = GUILayout.Window(0, discordWindowRect, DoDiscordWindow, "Server join request from Discord");
+        }
+
+        private void SetGUIStyle()
+        {
+            GUI.skin.textArea.fontSize = 18;
```

Comment:
This will overwrite the default IMGUI skin. You should make a copy of this skin and then change it.

```CS
private GUISkin SetGUIStyle()
{
    return GUISkinUtils.RegisterDerivedOnce("menus.discord", s =>
    {
        s.textArea.fontSize = 18;
        s.textArea.richText = false;
        s.textArea.alignment = TextAnchor.MiddleLeft;
        s.textArea.wordWrap = true;
        s.textArea.stretchHeight = true;
        s.textArea.padding = new RectOffset(10, 10, 5, 5);

        s.label.fontSize = 14;
        s.label.alignment = TextAnchor.MiddleRight;
        s.label.stretchHeight = true;
        s.label.fixedWidth = 140;
        s.label.fixedHeight = 140;

        s.button.fontSize = 14;
        s.button.stretchHeight = true;
    }
}
```